scripts/replace-in-regions.sh

```diff
@@ -0,0 +1,81 @@
+. $(dirname ${BASH_SOURCE[0]})/lib.sh
+
+# Replaces text using a sed pattern in the region of the file surrounded by
+# `region replace {region}` and `endregion replace {region}` comments.
+function replace_in_regions_for_file {
+    local file="$1"
+    local region="replace $2"
+    local sed_pattern="$3"
+
+    local comment_begin="(#|\/\/|<!--)"
+    local comment_end="( -->)?$"
+
+    # Replace both the version itself, and the sliding tags
+    #
+    # There is a caveat here for the major version. It is rather ambiguous, because
+    # it is just a single number, there are no dots in it that could identify it as
+    # semver version. So for the major version we require that it is always specified
+    # with the `v` prefix e.g. `v1`.
```

Comment:
This comment can be removed, since it applies to the function below, and it's already mentioned in the doc comment of that one

Reply:
Yeah, forgot to remove the comment, thanks!